### PR TITLE
Added shortcodes for minimum ksonnet and Kubernetes versions. (#168)

### PR DESCRIPTION
**Pull request by abhi-g**
Cherry picking Sarah's changes from #168 into v0.2.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/181)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,8 +15,219 @@ macOS...sorry :-)
 
 `hugo new doc/add-this-doc.md`
 
-### Making changes to CSS
+If you'd like to preview your doc updates as you work, you can install Hugo
```
Comment (sarahmaddox) [inline README.md]: This is only a part of the updates to the README file on the master branch. We should include all the updates, as this bit looks odd when placed immediately after the non-updated part.

File: content/docs/guides/requirements.md [hunk 2]
```diff
@@ -11,17 +11,14 @@ aliases = ["/docs/guides/"]
...
+* [TensorFlow](https://www.tensorflow.org/get_started/)
+* [ksonnet](https://ksonnet.io/docs/tutorial)
 
 For an end to end example illustrating in details how to deploy kubeflow and run a training job from scratch, check out this [tutorial](https://ai.intel.com/lets-flow-within-kubeflow/).
 
-## Requirements
- * Kubernetes >= 1.8 [see here](https://github.com/kubeflow/tf-operator#requirements)
- * ksonnet version [0.11.0](https://ksonnet.io/#get-started). (See [below](#why-kubeflow-uses-ksonnet) for an explanation of why we use ksonnet)
- * An existing kubernetes cluster:
+ * ksonnet version {{% ksonnet-min-version %}} or later. See the [ksonnet component page](/docs/guides/components/ksonnet/) for an explanation of why we use ksonnet.
```
Comment (sarahmaddox) [inline content/docs/guides/requirements.md]: Similar to my previous comment, this partial merge of content from the master branch doesn't work well here.

**Comment by abhi-g** [comment]
Okay, most likely that means there are a few other commits that I need to
pull in this PR. I’ll update this tomorrow.

Thanks
Abhishek

**Comment by abhi-g** [comment]
Abandoning this attempt. Recreating another PR to pull in relevant commits for completeness.

**Pull request closed without merging**